src/main/java/rotateit/github/domain/Label.java

```diff
@@ -0,0 +1,16 @@
+package rotateit.github.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public class Label {
+    private final String name;
+
+    @JsonCreator
+    public Label(String name) {
```

Comment:
argument is not annotated with `@JsonProperty` and there is no `jackson-modules-java8` in dependencies. I think jackson might have problems deserializing this
see: https://stackoverflow.com/questions/21920367/why-when-a-constructor-is-annotated-with-jsoncreator-its-arguments-must-be-ann (and first answer + comments)